src/map/unit.hpp

```diff
@@ -172,8 +172,8 @@ void unit_free_pc(map_session_data *sd);
 #define unit_remove_map(bl,clrtype) unit_remove_map_(bl,clrtype,__FILE__,__LINE__,__func__)
 int unit_remove_map_(struct block_list *bl, clr_type clrtype, const char* file, int line, const char* func);
 int unit_free(struct block_list *bl, clr_type clrtype);
-int unit_changeviewsize(struct block_list *bl,short size);
-int unit_changetarget(struct block_list *bl,va_list ap);
+int unit_changetarget(block_list *bl,va_list ap);
+void unit_change_target(unit_data& ud, int target_id);
```

Comment:
```suggestion
void unit_changetarget(unit_data& ud, block_list& target);
```
We use C++ now, function overloading is fine.